src/ElasticSearch.php

```diff
@@ -75,9 +88,13 @@ public function query($queryString, $type, $start = 0, $count = 10, $order = '_i
         }
 
         try {
+            $query = $this->getQueryBuilder()->getSearchParams($queryString, $type, $start, $count, $order, $dir);
+
+            $this->getLogger()->debug('Query ', $query);
+
             return $this->buildResultSet(
                 $this->getClient()->search(
-                    $this->getSearchParams($queryString, $type, $start, $count, $order, $dir)
+                    $query
                 )
             );
         } catch (\Exception $e) {
```

Comment:
I think that it would be good at least to add a log here if something wrong happens.

Reply:
Thanks. Fixed